### DISCUSSION TITLE
Request: Platform Overlay Plugin

### DISCUSSION THREAD

**Discussion by Rumik** [SteamDeckHomebrew/decky-loader#554] [Ideas]
Hi there!

Just wondering if anyone had thought of making a plugin that allows users to specify a platform for a given game and therefore display an appropriate icon over the top of the game grid artwork? I would find that immensely useful! I picture this working similarly to the CSS Loader plugin "show install status".

Use case: I have locally installed PC games, but I also have emulated Switch games too. And I use PlayStation Remote Play and Xbox Remote Play/Game Pass. It would be amazing to be able to include icons for each of these in the corner of the game grid artwork.

Thanks for reading!

**Comment by TrainDoctor** [comment]
Your interest is appreciated but we don't allow for plugin requests on decky loader repos issues etc. Thanks.
